lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -48,6 +48,12 @@ def setup_rabbitmq_connection_config
         # Time in seconds to wait before retrying a connection
         config :connect_retry_interval, :validate => :number, :default => 1
 
+        # The default connection timeout; zero means wait indefinitely
+        config :connection_timeout, :validate => :number
+
+        # Heartbeat delay in seconds. Default is 0 (no heartbeats)
+        config :heartbeat, :validate => :number, :default => 0
```

Comment:
Minor opinion: I don't like C-style making special numbers mean special things (0 means "no heartbeat at all" and not "no delay between heartbeats"). In Logstash, we could have no default value (remove `:default => 0` and document "If not given, heartbeating is disabled") and have the same behavior but without redefining "0 delay means disabled"

Thoughts?